src/awkward/_v2/contents/listarray.py

```diff
@@ -959,6 +959,92 @@ def _validityerror(self, path):
             else:
                 return self.content.validityerror(path + ".content")
 
+    def _rpad(self, target, axis, depth, clip):
+        if not clip:
+            posaxis = self.axis_wrap_if_negative(axis)
+            if posaxis == depth:
+                return self.rpad_axis0(target, clip)
+            elif posaxis == depth + 1:
+                # min = ak._v2.index.Index64.empty(1, self.nplike)
+                # self._handle_error(
+                #     self.nplike[
+                #         "awkward_ListArray_min_range",
+                #         min.dtype.type,
+                #         self._starts.dtype.type,
+                #         self._stops.dtype.type,
+                #     ](
+                #         min.to(self.nplike),
+                #         self._starts.to(self.nplike),
+                #         self._stops.to(self.nplike),
+                #         len(self._starts),
+                #     )
+                # )
+                min_ = self.nplike.min(self._stops.data - self._starts.data)
```

Comment:
Same as in a previous comment. It's great to replace kernels with NumPy functions. Let's also get rid of the commented-out code. If you're thinking about the problem of finding unused kernels (knowing, for instance, that `awkward_ListArray_min_range` is no longer used and can be removed), I plan on making a sweep to search for all of those in bulk, very much like "garbage collection," but for dead code. For the same reason as garbage collection of unreferenced data, it will be easier to do that in bulk.

Reply:
My thinking was that it would be easier to catch commented out kernels later, and remove their implementations permanently. But as you suggested, an automated grep search could also help catch every unused kernel. I have removed the commented out code in the latest commit.